docs/developer/running-saleor/exposing-instance.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Exposing local instance
+sidebar_label: Exposing instance
+---
+
+There are situations when you would like to expose your local environment to external clients, for example:
+
+- show your project to the manager.
+- integrate with the new payment gateway, which requires a return URL.
```

Comment:
This won't work. There is a settings.ALLOWED_CLIENT_HOSTS which defines where user can be redirected. If you don't overwrite this variable, the allowed redirect URL will be 127.0.0.1 and localhost. Any different than this will raise an error. The same case is for the situation when you're hosting storefront somewhere, and you use ngrok to connect it to your backend (all mutation like, reset password, send activation URL will raise an exception, afair).